ggml/include/ggml.h

```diff
@@ -2010,10 +2036,23 @@ extern "C" {
     GGML_API size_t ggml_graph_overhead(void);
     GGML_API size_t ggml_graph_overhead_custom(size_t size, bool grads);
 
+    GGML_API struct ggml_threadpool_params   ggml_threadpool_params_default(int n_threads);
+    GGML_API void                            ggml_threadpool_params_init(struct ggml_threadpool_params *p, int n_threads);
+    GGML_API bool                            ggml_threadpool_params_match (const struct ggml_threadpool_params *p0, const struct ggml_threadpool_params *p1);
+    GGML_API struct ggml_threadpool*         ggml_threadpool_create       (struct ggml_threadpool_params  * params);
+    GGML_API void                            ggml_threadpool_release      (struct ggml_threadpool * threadpool);
```

Comment:
Let's rename these to `_new` (or `_init`) and `_free` to be a bit more consistent with other functions in the ggml API.

Reply:
done